templates/theme-icon.php

```diff
@@ -0,0 +1,88 @@
+<?php
```

Comment:
@fajardoleo Same about theme-icon.php, it doesn't make sense to have almost the same code twice (unless there are too many differences).

When you adjust the plugin-icon.php, look at issue #67, and adjust the logic based on the suggestion. If it's not clear how to do it, LMK